src/winetricks

```diff
@@ -16895,6 +16895,21 @@ _EOF_
 
 #----------------------------------------------------------------
 
+w_metadata procexp apps \
+    title="Process Explorer" \
+    publisher="Steve P. Miller" \
+    year="2006" \
+    media="download" \
+
+load_procexp()
+{
+    w_download https://download.sysinternals.com/files/ProcessExplorer.zip c50bddaaacb26c5654f845962f9ee34db6ce26b62f94a03bb59f3b5a6eea1922
+    w_try_unzip "${W_TMP}" "${W_CACHE}"/procexp/ProcessExplorer.zip
+    w_try cp "${W_TMP}"/procexp*.exe "${W_DRIVE_C}/windows"
```

Comment:
This will install the 64bit exe in a 32-bit prefix.

Also, please use `${W_WINDIR_UNIX}` here.

Reply:
The issues has been fixed - thanks for reviewing.